ot3_state_manager/generate_state_manager_headers.py

```diff
@@ -0,0 +1,66 @@
+"""Script to generate hpp file with OT3StateManager message enums."""
+
+import argparse
+import sys
+from enum import Enum
+from typing import Type
+
+from header_generation_utils.utils import generate_file_comment, write_enum_cpp
+
+from ot3_state_manager.messages import MessageID
+from ot3_state_manager.util import Direction, MoveMessageHardware, SyncPinState
+
+
+def get_message_id_enum() -> Type[Enum]:
+    """Creates a MessageID enum."""
+    message_ids = [
+        ("_".join([message.name, "message_id"]).lower(), message.message_id)
```

Comment:
Are you meaning turn it into something like 
```
enum class MessageID {
    move = 0x0,
    sync_pin = 0x1,
    get_axis_location = 0x2,
    get_sync_pin_state = 0x3,
};
```
instead of 
```
enum class MessageID {
    move_message_id = 0x0,
    sync_pin_message_id = 0x1,
    get_axis_location_message_id = 0x2,
    get_sync_pin_state_message_id = 0x3,
};
```
because the `message_id` part is redundant?